cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -39,6 +39,9 @@ fi
 Content-Type: text/cloud-config; charset=us-ascii
 MIME-Version: 1.0
 
+bootcmd:
+  - if [ "${DisableHyperThreadingManually}" = "true" ]; then for cpunum in $(cat /sys/devices/system/cpu/cpu*/topology/thread_siblings_list | tr '-' ',' | cut -s -d, -f2- | tr ',' '\n' | sort -un); do echo 0 > /sys/devices/system/cpu/cpu$cpunum/online; done; fi
```

Comment:
From the tests I have seen that you checked with Intel CPU. Do you check this with AMD?
It is not possible to get here with `DisableHyperThreadingManually=true` and ARM, am I recalling right?

Reply:
AMD case it's covered as well, e.g. c5a.xlarge
```
# cat /sys/devices/system/cpu/cpu*/topology/thread_siblings_list
0,2
1,3
0,2
1,3
```

ARM case, `disable_simultaneous_multithreading_manually` is false because this condition `self.instance_type_info.default_threads_per_core() > 1` is not verified, nevertheless code still works because
output of the cat will be empty